### DISCUSSION TITLE
/car command

### DISCUSSION THREAD

**Discussion by jcaspes** [sinipelto/dayz-scripts#2] [Q&A] [answered]
Hello,
Thank for you scripts !
I've a question on /car command, when i use it, a vehicule spawn correctly but if i logoff / relog, the car is destroyed, no more present.
Do we need to do some economy modificaiton to be able to make spawned cars with /car command to be persistent ?

Thank you !

**Comment by sinipelto** [comment]
Hi!
I believe you would need to have persistence configured and enabled on your server to make the objects retain. What I have experienced, the vehicles spawned using the /car command should persist on the server if the persistence is enabled, and the command itself should not affect the persistency.

**Reply by jcaspes** [reply]
Hey, thank for your answer.
Persistence is enabled on my server, all found cars on mapo persits.
I'm on Experimental version on Livonia map
I can see that in types.xml file, nominal value for cars is set to 0 .. i try to set it to a bigger value to see if economy accept a car

**Reply by jcaspes** [reply, marked as answer]
Hello, I found the root cause, it was not the type.xml nominal value that was a problem but the lifetime that is set to 3 (3 seconds) on this version
setting a lifetime to 86400 on the OffroadHatchback type make the car not despawning before 24 hours
